hpedockerplugin/file_manager.py

```diff
@@ -549,7 +551,21 @@ def mount_share(self, share_name, share, mount_id):
         if 'status' in share:
             if share['status'] == 'FAILED':
                 LOG.error("Share not present")
-
+        fUser = None
+        fGroup = None
+        fMode = None
+        fUName = None
+        fGName = None
+        is_first_call = False
+        if share['fsOwner']:
+            fOwner = share['fsOwner'].split(':')
+            fUser = int(fOwner[0])
+            fGroup = int(fOwner[1])
+        if share['fsMode']:
```

Comment:
Is it must for the user to specify fsMode when fsOwner is specified? If not, then we should be doing share.get('fsMode') here otherwise we will get KeyError.

Reply:
this will have None value for fsMode. We are handling it in request_context